week-2/F-logical-operators/exercise2.js

```diff
@@ -5,17 +5,29 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(number) {
+  return number <0 ? true : false;
```

Comment:
Good use of ternary operators. well done.

Reply:
Seconding Kadir's point here. I know one day we will all have to learn how to do these in our everyday code but it still makes no sense to me! Very impressed at how clean it makes everything look.